airbyte-integrations/connectors/source-s3/source_s3/stream.py

```diff
@@ -55,12 +55,12 @@ def _list_bucket(self, accept_key=lambda k: True) -> Iterator[str]:
                 for c in content:
                     key = c["Key"]
                     if accept_key(key):
-                        yield key
+                        yield FileInfo(key=key, last_modified=c["LastModified"], size=c["Size"])
```

Comment:
Like this, means we can grab all the useful information in one sweep rather than doing more network calls.

Could you make sure all relevant docstrings (e.g. on abstract filepath_iterator()) are updated to reflect where LastModified should now be coming from.

Reply:
checked and updated